src/Controller/SoapController.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * @file
+ * Controller to expose SOAP endpoint that mimics the original 'moreInfo' service.
+ */
+
+namespace App\Controller;
+
+use App\Service\MoreInfoService\MoreInfoService;
+use Psr\Log\LoggerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class SoapController extends AbstractController
+{
+    /**
+     * @Route("/2.11/", name="soap_trailing_slash")
+     */
+    public function soap(Request $request, MoreInfoService $moreInfoService, LoggerInterface $statsLogger, $projectDir)
+    {
+        $soapServer = new \SoapServer($projectDir.'/src/Service/MoreInfoService/Schemas/moreInfoService.wsdl', ['cache_wsdl' => WSDL_CACHE_NONE]);
```

Comment:
Should be spaces between string concatenations.

Reply:
Not according to our CS FIX config....

Reply:
okay, just looks so wrong in my eyes :-|